src/azure-cli/azure/cli/command_modules/rdbms/validators.py

```diff
@@ -116,6 +116,22 @@ def validate_private_endpoint_connection_id(cmd, namespace):
     del namespace.connection_id
 
 
+def pg_storage_validator(ns):
+    if ns.storage_mb:
+        if str(ns.storage_mb) in ['32', '64', '128', '256', '512', '1024', '2048', '4096', '8192', '16384']:
```

Comment:
Ideally this should come from the capabilities API call, and i think we already have this..

Reply:
Added as task in onenote as API may be pending